ts/initializeTodos.ts

```diff
@@ -0,0 +1,22 @@
+import { todoModel } from "./todoModel.js";
+
+function initializeTodos() {
```

Comment:
Perfect 👍 Just compare the previous variant and the current one. Simplicity is great